.azure-pipelines/build-job.yml

```diff
@@ -138,6 +138,7 @@ jobs:
     - script: ${{ variables.devCommand }} testl0 Debug ${{ parameters.os }}-${{ parameters.arch }}
       workingDirectory: src
       displayName: Unit tests
+      timeoutInMinutes: 5
```

Comment:
Is this change needed?

Reply:
Yes, either unit tests pass quickly, or they hang for an hour and fall.